pages/index.mdx

```diff
@@ -30,15 +31,31 @@ The GOV.UK PaaS team manages the underlying platform, so you can:
 
 <GridColumn width='two-thirds'>
 
+## Save your team time and money
+
+By using GOV.UK PaaS you may save money by:
+
+-   avoiding the upfront costs of being forced to build or procure your own infrastructure or platform
+-   avoiding daily maintenance costs from needing to employ infrastructure engineers to keep services securely patched, up to date and running efficiently
+-   needing less information assurance because the platform is consistent and assured by Cabinet Office
```

Comment:
I'm not sure what `platform is consistent` is trying to say here.

Reply:
"needing less information assurance because the platform is accredited and assured by Cabinet Office"

?

Reply:
sounds good